src/interface/variable_state.hpp

```diff
@@ -61,7 +61,10 @@ struct VariableState : public empty_state_t {
   int sparse_id;
   int vector_component = NODIR;
   bool initialized = true;
-
+  
+  int t{0};
+  int u{0};
+  int v{0};
```

Comment:
Maybe should be initialized to some signal var indicating they haven't been set, like -1?

Reply:
:+1: on the default init.

Unrelated to this PR and a more general performance question: Has anyone looked at the impact on performance when putting more and more info in the packs?

Reply:
yeah, probably so. Also maybe would be better to store as an array? Not totally sure.

Reply:
The message from @pgrete didn't show up for me before. I have not looked carefully at the performance impact, but at least when I put variable state in initially I didn't see any slow down. Of course, there is now more state stored there...

Reply:
If this is a performance issue, there are definitely things that can be removed and some things that can be moved to the sparse pack itself.